scrapers/GymRotic.yml

```diff
@@ -0,0 +1,41 @@
+name: DollRotic
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - dollrotic.com/trailers/
+      - femlatex.com/trailers/
+      - gymnastic.xxx/trailers/
+      - gymrotic.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //li[@class="active"]
+      Date:
+        selector: //strong[contains(text(),"Released:")]/following-sibling::text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //strong[contains(text(),"Featuring:")]/following-sibling::a
+      Tags:
+        Name: //strong[contains(text(),"Tags:")]/following-sibling::a
+      Details:
+        selector: //div[@class="col-md-7 mb-4 mb-md-0"]//p
+      Image:
+        selector: //base/@href|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
+        concat: "|"
+        postProcess:
+          - replace:
+              - regex: "[|]"
+                with: ""
+      Studio:
+        Name:
+          selector: //base/@href
+          postProcess:
+            - map:
+                https://www.dollrotic.com/: DollRotic
+                https://www.gymnastic.xxx/: GymnasticXXX
+                https://www.femlatex.com/: FemLatex
+                https://www.gymrotic.com/: GymRotic
```

Comment:
depending on how you access the site you can end up with a `https://www.xxxx.com` or a `https://xxxxx.com` url
i would just use the author meta and do a simple regex replace eg
```yaml
      Studio:
        Name:
          selector: //meta[@name="author"]/@content
          postProcess:
            - replace:
                - regex: '\.com$'
                  with: ""
```
you might want to add an extra replace/map for the gymnastic site if you like the GymnasticXXX vs the  Gymnastic.xxx you will get from the proposed change

Reply:
Nar I am not worried about it. :-)